src/api/two/models.rs

```diff
@@ -19,7 +19,7 @@ pub struct Meta {
     pub next: Option<String>,
     pub previous: Option<String>,
     #[serde(rename = "self")]
-    pub _self_: String,
+    pub self_url: String,
     pub first: String,
     pub last: Option<String>,
```

Comment:
Is there a reason to use `Option` here, but not on `first`?

Reply:
I remember only `last` being null or missing in one case. If that is also the case for `first` then it would make sense to also make it optional. (But I see you changed a `fixture`, maybe the optional is superfluous now).

Reply:
I've found a real world example (http://167.114.43.38:4003/api/blocks/12029645780492672346/transactions) with a `null` value.

IMHO it's confusing, because, when there is only 1 page, first page and last are the same, so both should have value, but maybe is working that way on purpose.